content/03.methods.md

```diff
@@ -4,17 +4,44 @@
 
 ### Pediatric Pacific Neuro-oncology Consortium Data
 
-### DNA Alignment
+### DNA WGS Alignment
+
+We used BWA-MEM v0.7.17 [@arxiv:1303.3997] for alignment of paired-end DNA-seq reads.
+The alignment reference that we used was Homo Sapiens Human Genome (hg) hg38 fasta file obtanied from [UCSC](@http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/).
+Alignments were futher processed using following [Broad's Best Practices](@https://software.broadinstitute.org/gatk/best-practices/workflow?id=11165) for processing bams in preparation for variant discovery.
+Duplicates were marked using [Samblaster](@doi:10/f6kft3) v0.1.24, bams merged and sorted using [Sambamba](@https://lomereiter.github.io/sambamba/) v0.6.3.
```

Comment:
```suggestion
Duplicates were marked using Samblaster[@doi:10/f6kft3] v0.1.24, bams merged and sorted using Sambamba[@doi:10.1093/bioinformatics/btv098] v0.6.3.
```
It's preferred to use DOIs, PMIDs, etc before URLs if possible because the reference information can be automatically extracted. See the format changes above, which you'll need to make throughout to be compliant with the manubot citation engine.